app/views/projects/index.html.erb

```diff
@@ -3,7 +3,12 @@
 <div class="unvoted">
   <h1>Unvoted</h1>
   <% @projects.check_votes.each do |project| %>
-    <p><%= link_to project.name, new_project_vote_path(project) %></p>
+    <p>
+      <%= link_to "Vote for #{project.name}", new_project_vote_path(project) %>
+      <% if owner_and_accepting(project.id) %>
```

Comment:
So we show all projects, but only allow them to edit their own? And they're all listed as `Unvoted`? This would be a major overhaul, but I'm starting to think we should create namespaces for `accepting_submissions` and `voting`, and just have completely separate views/controllers. Maybe next time?